messages/en/app.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 return [
-    'kickstarter_success' => '<h1>You are done!</h1>
+    'kickstarter_success' => '<h1 class="alert-heading">You are ready!</h1>
     <p>Yes, you have successful installed <i>LUYA</i> on your System. You are now able to login into the administration area to create your pages.</p>
     <p><strong>Important</strong><br />After loging into the administration area you have to navigation to &laquo;System&raquo; and enable all permissions for your group (Adminstrator).</p>',
```

Comment:
"loging into" -> "logging into"
"to navigation to" -> "to navigate to"

Reply:
spelling and messages tide up will come in the next pr

Reply:
merge?

Reply:
Why not just do it right the first time?

Reply:
coz messages need top be reranged imo

Reply:
... and this PR is already 'full and a bit too much changes'

Reply:
I'm not sure what you tried to say, but just remember my suggestions for when you fix those mistakes. 👍

Reply:
🥇  i ll do